aiida/tools/importexport/archive/common.py

```diff
@@ -61,3 +74,144 @@ def detect_archive_type(in_path: str) -> str:
         'Unable to detect the input file format, it is neither a '
         'folder, tar file, nor a (possibly compressed) zip file.'
     )
+
+
+def read_file_in_zip(filepath: str, path: str) -> str:
```

Comment:
this returns the file contents as a string, right (i.e. it shouldn't be used for binary files)?

I think this should be mentioned in the docstring.
Ideally also in the function name but perhaps that would get too long...

Reply:
sure can add it to the function string, but I think the typing makes it quite clear

Reply:
done